CMakeLists.txt

```diff
@@ -82,7 +89,7 @@ OPTION(ENABLE_PACKAGING
   OFF)
 OPTION(BUILD_STATIC_LIBRARIES
   "create targets for building static libraries"
-  OFF)
+  ON)
```

Comment:
use -DBUILD_STATIC_LIBRARIES to turn on it, no need to change the default value.